canonical-web-and-design/microk8s.io
# Add Contact us

## Done

Add contact-us and thank-you page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- copy docs for [contact-us](https://drive.google.com/open?id=1KkLHMrGipY4zgIXreujpyJlLvYUs-iAlP-FtpYrZmWA) and [thank-you](https://drive.google.com/open?id=1zAmRxMVFQhUPv9jBjVHJPnWfJMjiB78dQfY__h4pE7o) pages


## Issue / Card
